src/main/java/com/umc/withme/domain/Point.java

```diff
@@ -0,0 +1,54 @@
+package com.umc.withme.domain;
+
+import com.umc.withme.domain.common.BaseTimeEntity;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+@Entity
+public class Point extends BaseTimeEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "point_id")
+    private Long id;
+
+    @Column(nullable = false)
+    private Integer attendance;
+
+    @Column(nullable = false)
+    private Integer passion;
+
+    @Column(nullable = false)
+    private Integer contact_speed;
```

Comment:
```suggestion
    @Column(nullable = false)
    private Integer contactSpeed;
```
이거 java 변수명이니까 camel-case로 작성해주세요!